src/App.jsx

```diff
@@ -14,33 +15,87 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [list, setList] = useState(goodsFromServer);
```

Comment:
you don't need the separate state for the list in this task

Reply:
```suggestion
  const visibleGoods = getVisibleGoods(goodsFromServer, sortType, isReversed)
```
and move all the sorting and reversing logic into the getVisibleGoods function